modules/core/test/v2/unit/coins/btc.ts

```diff
@@ -320,20 +321,20 @@ describe('BTC:', function () {
         },
       };
 
-      let coin;
+      let coin: AbstractUtxoCoin;
```

Comment:
I'm curious why you specified a type here if you have to cast to `any` below?

Reply:
to get IntelliJ support :)

Reply:
the tests where I have to cast to `any` are simulating use from untyped JS